woocommerce/woocommerce-gutenberg-products-block
# [Product Collection] Fix: HTML Entity Decoding in Taxonomy Controls

## What

Fix HTML entity decoding issue in Taxonomy filters. You can see the issue in screenshot below:
<img width="271" alt="image" src="https://github.com/woocommerce/woocommerce-blocks/assets/16707866/80e30d1f-cbb0-4182-9272-f6328467fc89">



Fixes #9242 

## Why




## Testing Instructions



_Please consider any edge cases this change may have, and also other areas of the product this may impact._

1. Create a new post & add Product Collection block to it.
2. From filters dropdown, Add “Taxonomies” filter. This will add “Product Categories” and “Product Tag” taxonomy filters to inspector control.
3. Inside “Product Categories” filter, search a category that contains an HTML Entity.
    1. If you don’t have any category with HTML entities, then you will need to either rename the existing category or create a new category with HTML entities. For example, “Electronics & Gadgets” or “Health & Wellness / Supplements”
    2. Verify that HTML entities show as expected. 
4. Perform step 3 for “Product Tags” too.


## Screenshots or screencast



| Before | After |
| ------ | ----- |
|        |       |

## WooCommerce Visibility


Required:


## Checklist

Required:


Conditional:


## Changelog


> Product Collection - Fix: HTML Entity Decoding in Taxonomy Controls
